.github/workflows/dotnet.yml

```diff
@@ -35,8 +42,13 @@ jobs:
       run: dotnet paket restore
     - name: Build and Test
       run: dotnet fake run build.fsx
-    - name: Publish NuGet
-      if: matrix.os == 'windows-latest' && github.ref == 'refs/heads/master'
+    - name: Publish prerelease NuGet to GitHub
+      if: matrix.os == 'windows-latest' && github.ref == 'refs/heads/master' && (github.event.inputs.release == null || github.event.inputs.release == 'false')
       run: dotnet fake run build.fsx -t PublishCINuGet
       env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Publish NuGet to NuGet.org
+      if: matrix.os == 'windows-latest' && github.ref == 'refs/heads/master' && github.event.inputs.release == 'true'
+      run: dotnet fake run build.fsx -t Release
```

Comment:
this call will execute all FAKE targets from the beginning and probably fail on `CINuGet` because we do not provide `GITHUB_TOKEN`.

You may want to merge `"NuGet" ==> "PublishNuget"` into one FAKE target and then execute with `--signle-targe` param, like `dotnet fake run build.fsx -t "PublishNuget" --single-target`

Reply:
I **think** that `CINuGet` only prepares the package while `PublishCINuGet`  requires the GITHUB_TOKEN. From what I see `Release` does not depend on `PublishCINuGet`. We will see in a moment :)

Reply:
I ddin't know about `--single-target` - noted.